homeassistant/components/homekit/type_thermostats.py

```diff
@@ -175,7 +175,7 @@ def __init__(self, *args):
 
         self.char_target_heat_cool = serv_thermostat.configure_char(
             CHAR_TARGET_HEATING_COOLING,
-            value=0,
+            value=list(hc_valid_values.values())[0],
```

Comment:
Are we sure that `hc_valid_values` always have content?
If not would `list(hc_valid_values.values())[0] if hc_valid_values else 0` be a safer option?

Reply:
If they don’t implement hvac_modes or have an empty list we likely need to not export the characteristic to HomeKit.   

Will need to do some testing to see how that works.

Reply:
Actually there is already code to handle this on line 139.